Hymns/Home/LaunchRouterView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+struct LaunchRouterView: View {
+    @State var showSplash: Bool = false
+
+    var body: some View {
+        Group { () -> AnyView in
+            if showSplash {
+                return SplashScreenView()
+                    .opacity(showSplash ? 1 : 0)
+                    .onAppear {
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 2.7) {
+                            self.showSplash = false
+                        }
```

Comment:
Is it possible to use `SplashScreenView`'s coordinator to determine when the animation is finished instead of waiting 2.7 seconds? I don't think there's any guarantee that the animation will run in 2.7 seconds, is there?

Reply:
Yes, there is. 2.7 seconds is the amount of time that it always takes to run. It should never change. and is set. Every-time, I have tested it finishes right on time.

Reply:
But I feel like when we test, it's in a rather controlled environment, with the same amount of ram, with all other apps cleared, and on a simulator/same device over and over again. Whereas in the wild, things will be quicker and slower depending on a number of factors (https://www.knowyourmobile.com/user-guides/4-ways-to-speed-up-slow-iphone/)

I guess if it's not possible to create a listener with a Coordinator, then I'd be okay with keeping the 2.7 seconds, since this isn't a huge deal anyway. But just want to make sure that we have to do it this way before hardcoding a magic number (https://en.wikipedia.org/wiki/Magic_number_%28programming%29) in here